docs/app/pages/components/sections/select/select/snippets/app.snippet.ts

```diff
@@ -48,18 +48,19 @@ export class AppComponent implements OnInit {
         this.pagingEnabled.subscribe((value) => {
             this.selected = null;
             this.dropdownOpen = false;
-            this.options = this.pagingEnabled.getValue() ?
-                this.loadOptionsCallback :
-                this.selectedDataSet();
+            this.options = this.pagingEnabled.getValue() ? this.loadOptionsCallback : this.selectedDataSet();
```

Comment:
What does this change accomplish?

Reply:
I don't think this accomplishes anything other than consistency in the layout of the code.  What I think has happened here is that I made changes in the flat select and rather than make them again individually, I copied the contents of the component over.  Sorry about that.

Reply:
In most cases, wrapping is to avoid a horizontal scrollbar in the code snippet window. I noticed that this did add a scrollbar, although it was only for the sake of a few pixels.